docs/paired_ttest.ipynb

```diff
@@ -0,0 +1,234 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This notebook shows how to use the switchback module. In particular, it shows how to create a PowerAnalysis object with a switchback splitter, using a time column and 30 min splits.\n",
```

Comment:
can you remove this? I don't think it applies. You also need to run the notebook if you want to see the results of the notebook in the docs

Reply:
you also need to add the notebook path in mkdocs.yml if you'd like to see it in the docs. You can type `make docs-serve` if you want to test the docs output locally